token-account-based/README.md

```diff
@@ -0,0 +1,200 @@
+- The same minter client will then use the `Transfer` function to transfer the requested number of tokens to the recipient's account. It is assumed that the recipient has provided their client ID to the transfer caller out of band. The recipient can then transfer tokens to other registered users in the same fashion.
+
+## Bring up the test network
+
+You can run the account-based token transfer scenario using the Fabric test network. Open a command terminal and navigate to the test network directory in your local clone of the `fabric-samples`. We will operate from the `test-network` directory for the remainder of the tutorial.
+```
+cd fabric-samples/test-network
+```
+
+Run the following command to start the test network:
+```
+./network.sh up createChannel -ca
+```
+
+The test network is deployed with two peer organizations. The `createChannel` flag deploys the network with a single channel named `mychannel` with Org1 and Org2 as channel members.
+The -ca flag is used to deploy the network using certificate authorities. This allows you to use each organization's CA to register and enroll new users for this tutorial.
+
+## Deploy the smart contract to the channel
+
+You can use the test network script to deploy the account-based token contract to the channel that was just created. Deploy the smart contract to `mychannel` using the following command:
+```
+./network.sh deployCC -ccn token_account -ccp ../token-account-based/chaincode-go/
+```
+
+The above command deploys the go chaincode with short name `token_account`. The smart contract will utilize the default endorsement policy of majority of channel members.
+Since the channel has two members, this implies that we'll need to get peer endorsements from 2 out of the 2 channel members.
+
+Now you are ready to call the deployed smart contract via peer CLI calls. But let's first create the client identities for our scenario.
+
+## Register identities
+
+The smart contract supports accounts owned by individual client identities from organizations that are members of the channel. In our scenario, the minter of the tokens will be a member of Org1, while the recipient will belong to Org2. To highlight the connection between the `GetClientIdentity().GetID()` API and the information within a user's certificate, we will register two new identities using the Org1 and Org2 Certificate Authorities (CA's), and then use the CA's to generate each identity's certificate and private key.
+
+First, we need to set the following environment variables to use the the Fabric CA client (and subsequent commands).
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+export FABRIC_CFG_PATH=$PWD/../config/
+```
+
+We will use the Org1 CA to create the minter identity. Set the Fabric CA client home to the MSP of the Org1 CA admin (this identity was generated by the test network script):
```

Comment:
It would provide more set up work, but would it be worth it to add another recipient from org2 and allow recipient 1 to transfer to some tokens to recipient 2. It would further demonstrate that the token is being held at the individual level, and not the org level.

Reply:
Sure, will do in another PR.